R/rd.R

```diff
@@ -243,7 +243,10 @@ topic_add_simple_tags <- function(topic, block) {
   tag_names <- names(block)[is_simple]
 
   for (i in seq_along(tag_values)) {
-    topic$add_simple_field(tag_names[[i]], tag_values[[i]])
+    topic$add_simple_field(tag_names[[i]], tag_values[[i]][1])
```

Comment:
```suggestion
    topic$add_simple_field(tag_names[[i]], tag_values[[i]][[1]])
```

Reply:
`tag_values[[i]]` is a character vector now, for tags with sections. This code is not the easiest to read, so we can make sections more explicit if you want. Then we'll need a `topic_add_tags_with_sections()` function I guess.